kahuna/app/controllers/KahunaController.scala

```diff
@@ -24,6 +24,10 @@ class KahunaController(config: KahunaConfig, override val controllerComponents:
     ))
   }
 
+  def quotas = Action { req =>
```

Comment:
should we throw auth in here?

Reply:
Ah that's a good point. I thought Kahuna didn't have access to auth actions at all but it does after the refactor! I may be able to do the whole thing as a twirl template then